client/src/types/User.ts

```diff
@@ -1,9 +1,16 @@
-export interface User {
+export interface BaseUser {
     uuid: string;
-    admin: boolean;
     name: string;
 }
 
+export interface BaseUserWithID extends BaseUser {
+    haveID: boolean;
```

Comment:
remind me why we need this interface? For example, is it bad to group it with admin

Reply:
Honestly we just have multiple GraphQL queries and different ones want different user properties.  I think `Maybe` type might be better and we could combine them all